sqlite3_test.go

```diff
@@ -1513,6 +1513,120 @@ func TestPinger(t *testing.T) {
 	}
 }
 
+type preUpdateHookDataForTest struct {
```

Comment:
Please move this to a file named sqlite3_opt_preupdate_hook_test.go and set a build tag sqlite_preupdate_hook